### PR TITLE
Created new class and add method

### PR DESCRIPTION
**Pull request by samedkocyigit**

### CODE REVIEW

**Review by furkankocyigit — CHANGES_REQUESTED** [review]
Please sepearate your link list implementation and test class like we did for array and strings.
Also I see some warnings about nun-nullable fields, please fix them too.

File: InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            CustomLinkedListsMethods list=new CustomLinkedListsMethods();
+            list.add(15);
+            list.add(11);
+            list.add(12);   
+            list.add(13);
+
+            Assert.AreEqual(list.head.data, 15);
+            Assert.AreEqual(list.head.next.data, 11);
+        }
```
Comment (furkankocyigit) [inline InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs:21]: Please fix "dereference possible null reference" here.
Also, check other data as well '12,13, null'

check if its null after "13"

**Review by furkankocyigit — CHANGES_REQUESTED** [review]
Nice and clean solution overall. I just requested a single easy change in the test method.
Thanks!

File: InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs (hunk 1 above)
Comment (samedkocyigit) [reply InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs:21]: Assert.AreEqual(list.head.next.next.next.data, 13);
 Assert.AreEqual(list.head.next.next.next.next.next.data, null);
I tried the next data null or not but this test not passed. I can not figure out the problem. 

Also i can not fix "dereference possible null reference".

Comment (samedkocyigit) [reply InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs:21]: Assert.AreEqual(list.head.next.next.next.next.next.data, null);
Just this line not passed the test.

Comment (furkankocyigit) [reply InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs:21]: Please consider that you are checking the right thing. Should you check the 'data' of the last item or the item itself?

Comment (samedkocyigit) [reply InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs:25]: Alright i get it. This is the correct query pattern. Look here bbfa648702f99935f1ad1c331198ac16fd9bf78e

Comment (furkankocyigit) [reply InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs]: I refactored your class name -> 76c914d5a3468e9d9c193c9426b53eb3a1fe9f8a

**Review by furkankocyigit — APPROVED** [review]
Nice and clean solution.

I made small two commits, glance at them and then feel free to resolve the conversation and merge your changes.

Thank!

**Pull request merged**